editor/animation_track_editor.h

```diff
@@ -184,6 +188,7 @@ class AnimationTimelineEdit : public Range {
 	void set_track_edit(AnimationTrackEdit *p_track_edit);
 	void set_zoom(Range *p_zoom);
 	Range *get_zoom() const { return zoom; }
+	void _zoom_callback(float p_zoom_factor, Vector2 p_origin, Ref<InputEvent> p_event);
```

Comment:
Making the callback public is probably not a good idea, should I instead wrap it in a separate method like `on_zoom()` that is called in `_zoom_callback()`, that the outer classes then call?

Reply:
Either that or use `friend` for the classes that need it, the callback should remain private

Reply:
good point, thanks!